public/views/app.html

```diff
@@ -1,87 +1,177 @@
-</script>
-<style>input.ng-pristine.ng-valid, {background-color: red}</style>
-<link rel='stylesheet', type='text/css' href="./../css/old.css">
+  <header class="main-header">
 
-<div id="app-container" ng-controller="GameController">
-  <div id="menu-container">
-    <div id="logo">
-      <span id="first-word">Cards</span>
-      <span id="second-word"> for</span>
-      <span id="third-word">Humanity</span>
-      <!-- Search form for email-->
-      <form name="searchForm">
-        <input class="" type="text" placeholder="Search for players..." ng-model="user.email"/>
-        <button type="submit" ng-click="getUserInfo(user, searchForm)" class="btn btn-success">Submit</button>
-      </form>
-    </div>
+      <style>
+          .main-header {
+            position: relative;
+            z-index: 100;
+          }
+          .site-nav a {
+            color: black;
+          }
+          .main-header.solid{
+            background-color: #f1f1f1;
+          }
+          input.ng-pristine.ng-valid, {background-color: red}
+          h1, h2, h3, h5, h6 {
+            font-size: 150%;
+            font-weight: bolder;
+          }
+        </style>
 
-    <div id = 'abandon-game-button' ng-click="abandonGame()">
-        Abandon Game
-    </div>
-        <span id="tweet-container">
-      <a href="https://twitter.com/share" class="twitter-share-button" data-url="http://cfh.io" data-text="Cards for Humanity: A Game for Horrible People Desperately Trying to do Good" data-related="CFH_App">Tweet</a>
-    <script>!function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0],p=/^http:/.test(d.location)?'http':'https';if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src=p+'://platform.twitter.com/widgets.js';fjs.parentNode.insertBefore(js,fjs);}}(document, 'script', 'twitter-wjs');</script>
-    </span>
-  </div>
-  <div id="gameplay-container">
-    <div id="upper-gameplay-container">
-      <div id="menu-timeremaining-container">
-        <div id="menu-container">
-          <span id="menu-button">Menu</span>
+        
+
+      <div landing class="h-container" >
+  
+  
+        <div class="logo" ng-controller="IndexController">
+          <img src="/img/playing-cards.png" alt="">
```

Comment:
There are too many uncovered lines. Always ensure you don't leave lines uncovered except when you need to differentiate a section from another, it makes your code look better.